ajax/libs/angular-patternfly/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "angular-patternfly",
+  "author": "Red Hat",
+  "license": "Apache-2.0",
+  "filename": "angular-patternfly.min.js",
+  "keywords": [
+    "angularjs",
+    "patternfly"
+  ],
+  "description": "Angular extension of the PatternFly project.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/patternfly/angular-patternfly.git"
+  },
+  "homepage": "https://www.patternfly.org/angular-patternfly",
+  "npmName": "angular-patternfly",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "!(docs|less)/**/*",
+        "angular-paternfly*"
```

Comment:
This pattern can't get `angular-patternfly.js` & `angular-patternfly.min.js`
Need to modify from `angular-paternfly*` to `angular-patternfly*` (add a `t` --> angular-pat**t**ernfly*)
Thank you.